x-pack/legacy/plugins/ml/public/application/explorer/explorer_dashboard_service.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+/*
+ * Service for firing and registering for events across the different
+ * components in the Explorer dashboard.
+ */
+
+import { isEqual, pick } from 'lodash';
+
+import { from, isObservable, Observable, Subject } from 'rxjs';
+import { distinctUntilChanged, filter, flatMap, map, pairwise, scan, tap } from 'rxjs/operators';
+
+import { loadExplorerDataActionCreator } from './actions';
+import { explorerReducer, getExplorerDefaultState, ExplorerState } from './reducers';
+
+export const ALLOW_CELL_RANGE_SELECTION = true;
```

Comment:
looks like it's intended to be configurable?

Reply:
This is older code that was copied over from the `js` file, it's a bit like a hard coded feature flag. It's not meant to be configurable via settings but we've used it in the past to disable the feature in public versions because it was not stable enough.